pkg/upgrade/1.0.0-1.1.0/v1alpha1/spc_upgrade.go

```diff
@@ -72,6 +73,24 @@ func spcUpgrade(spcName, openebsNamespace string) (*utask.UpgradeTask, error) {
 			}
 		}
 	}
+	err = updateSPC(spcName)
+	if err != nil {
+		return nil, err
+	}
 	klog.Infof("Upgrade Successful for spc %s", spcName)
 	return nil, nil
 }
+
+func updateSPC(name string) error {
+	client := spc.NewKubeClient()
+	spcObj, err := client.Get(name, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	spcObj.VersionDetails.Desired = upgradeVersion
```

Comment:
why only Desired? We need to set for other fields as well?

Reply:
only desired is set by the job all other fields are populated by the cstor-operator itself.